wgpu-native/src/command/render.rs

```diff
@@ -320,6 +320,7 @@ pub extern "C" fn wgpu_render_pass_set_index_buffer(
 #[no_mangle]
 pub extern "C" fn wgpu_render_pass_set_vertex_buffers(
     pass_id: RenderPassId,
+    start_slot: u32,
```

Comment:
we still need to make it affect the `pass.vertex_state.inputs.iter_mut()` part, i.e. by doing `pass.vertex_state.inputs[start_slot..].iter_mut()`